chainguard-dev/edu
# adding Nathan's SSO/Custom IDP docs

## Type of change

This PR adds the Nathan's docs (that have been markdown-ified and feature some fancy new screenshots).

### What should this PR do?

resolves https://github.com/chainguard-dev/internal/issues/1627

### How should this PR be tested?

Make sure all the screenshots look okay - i felt like I was juggling a lot as I was transferring things from Google Drive to my IDE. Also give it a close copy edit, please.

@nsmith5 , I'd appreciate it if you gave things a high-level once over. Especially things like titles and step headings - I'm not totally sure the language I'm use to describe everything is totally right.
